client/src/main/java/sigblockchain/projectred/client/Account.java

```diff
@@ -0,0 +1,64 @@
+package sigblockchain.projectred.client;
+
+import java.math.BigInteger;
+
+public class Account {
+	private String WalletAddress;
+	private BigInteger Balance;
+	private BigInteger StateNonce;
+
+	public Account(String walletAddress, BigInteger balance, BigInteger stateNonce) {
+		validateWalletAddress(walletAddress);
+
+		WalletAddress = walletAddress;
+		Balance = balance;
+		StateNonce = stateNonce;
+	}
+
+	public Account(String walletAddress) {
+		validateWalletAddress(walletAddress);
+
+		WalletAddress = walletAddress;
+		Balance = BigInteger.ZERO;
+		StateNonce = BigInteger.ZERO;
+	}
+
+	private void validateWalletAddress(String walletAddress) {
```

Comment:
This seems to be the standard for bitcoin. Aurum does not follow those standards.